src/Error/index.js

```diff
@@ -1,18 +1,37 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-const Error = ({ error, marginTop = "xxs" }) => {
-  if (!error) return null;
+const ErrorLine = ({ errorLine, marginTop = "xxs" }) => {
+  if (!errorLine) return null;
   return (
     <div className={`nds-input-error margin--top--${marginTop}`}>
       <div className="fontSize--s margin--right--xxs narmi-icon-x-circle" />
-      {error}
+      {errorLine}
     </div>
   );
 };
+ErrorLine.propTypes = {
+  errorLine: PropTypes.string,
+  marginTop: PropTypes.oneOf(["xxs", "xs", "s", "m", "l", "xl", "xxl", "none"])
+};
+
+const Error = ({ error, marginTop = "xxs" }) => {
+  if (!error) return null;
+  if (Array.isArray(error)) {
+    return (
+      <div className="nds-input-errorlist">
+        {error.map((errorLine, index) => (
+          <ErrorLine key={index} errorLine={errorLine} marginTop={index ? marginTop : "xxs"} />
```

Comment:
nit: i feel like react yells at me if i use a mapped array index as the key "since it could change"

Reply:
oh you're right! I absolutely should have used the errorLine as the key, lemme fix that